src/components/hamburgerMenu.js

```diff
@@ -6,36 +6,47 @@ export default function HamburgerMenu({
   setMobileExpanded,
   mobileNavRef,
   width,
+  activeColor,
 }) {
   const [active, setActive] = useState(false)
-  const inactiveClass = "hamburger hamburger--slider"
-  const activeClass = "hamburger hamburger--slider is-active"
+  const inactiveClass = "hamburger"
+  const activeClass = "hamburger is-active"
 
   useEffect(() => {
     const handleResize = () => {
       if (width > 889 && width < 900) {
         setActive(false)
-        headerRef.current.classList.add("header--primary")
-        headerRef.current.classList.remove("header--secondary")
+        headerRef.current.classList.remove("white")
+        headerRef.current.classList.remove("magenta")
```

Comment:
You can do this in one line like `headerRef.current.classList.remove("white", "magenta")`

see: https://developer.mozilla.org/en-US/docs/Web/API/Element/classList